.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where is your package.json so I can run the project?
What are those file names? Get used to lower case on folder name and file names.
Read about Kebab-Case.

Reply:
Works in 2 tabs, but user names are not visible.